src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let rentalCost = days * 40;
+  const standartDiscount = 20;
+  const increasedDiscount = 50;
+
+  if (days >= 7) {
+    rentalCost = rentalCost - increasedDiscount;
```

Comment:
Improvement tip 💡 :

Instead of directly assigning the updated `rentalCost`, you can consider use shorter approach, to use '-='  `rentalCost -= increasedDiscount;`